sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/externalrules/CxxExternalRulesSensor.java

```diff
@@ -87,4 +119,48 @@ public void stream(SMHierarchicCursor rootCursor) throws XMLStreamException {
 
     parser.parse(report);
   }
+
+  private void transformFiles() throws TransformerException {
+    for(int i = 1; i < 100; i++) {
+      String stylesheetKey = SONAR_CXX_XSLT_KEY + i + STYLESHEET_KEY;
+      String sourceKey = SONAR_CXX_XSLT_KEY + i + SOURCE_KEY;
+      String outputKey = SONAR_CXX_XSLT_KEY + i + OUTPUT_KEY;
+
+      String stylesheet = FilenameUtils.normalize(settings.getString(SONAR_CXX_XSLT_KEY + i + STYLESHEET_KEY));
+      String[] sources = settings.getStringArray(SONAR_CXX_XSLT_KEY + i + SOURCE_KEY);
```

Comment:
We are doing this in CxxReportSensor.getReports. This is a static function, maybe we can move it https://github.com/SonarOpenCommunity/sonar-cxx/tree/master/sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils to make it more general?

https://github.com/SonarOpenCommunity/sonar-cxx/blob/056fbb3fcad753e658d0cc0df6603edf63b25d21/sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java#L126